docs/Homebrew-homebrew-core-Merge-Checklist.md

```diff
@@ -54,7 +54,7 @@ Check for:
 - if CI is green and...
   - formula `bottle :unneeded`, you can merge it through GitHub UI
   - bottles need to be pulled, and...
-    - the commits are correct and don't need changes, use: `brew pr-publish $PR_ID`
+    - the commits are correct and don't need changes, approve the PR to trigger an automatic merge (use `brew pr-publish $PR_ID` for new formulae to trigger manually)
```

Comment:
Since it's not just new formula I think this might be better

```suggestion
    - the commits are correct and don't need changes, approve the PR to trigger an automatic merge (use `brew pr-publish $PR_ID` to trigger manually in case of a new formula)
```